SiaInterface/bin/Renter.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:http/http.dart' as http;
+import 'dart:io';
+
+class Renter {
```

Comment:
All the methods in this class should be static so that I can access them using `Renter.download(...)` etc.